setup.cfg

```diff
@@ -20,3 +20,9 @@ versionfile_build = tableauserverclient/_version.py
 tag_prefix = v
 #parentdir_prefix =
 
+[aliases]
+smoke=pytest
```

Comment:
Yup. I have smoke running both the test directory and the smoke directory.  Test will only run the test directory